pkg/cmd/template/schema_author_test.go

```diff
@@ -756,6 +756,198 @@ schema.yml:
 				files.MustNewFileFromSource(files.NewBytesSource("schema.yml", []byte(schemaYAML))),
 			})
 
+			assertFails(t, filesToProcess, expectedErr, opts)
+		})
+	})
+	t.Run("when schema/examples annotation value", func(t *testing.T) {
+		t.Run("is empty", func(t *testing.T) {
+			schemaYAML := `#@data/values-schema
+---
+#@schema/examples
+key: val
+`
+			expectedErr := `
+Invalid schema
+==============
+
+syntax error in @schema/examples annotation
+schema.yml:
+    |
+  3 | #@schema/examples
+  4 | key: val
+    |
+
+    = found: missing value in @schema/examples (by schema.yml:3)
+    = expected: tuple with description and example
+`
+
+			filesToProcess := files.NewSortedFiles([]*files.File{
+				files.MustNewFileFromSource(files.NewBytesSource("schema.yml", []byte(schemaYAML))),
+			})
+
+			assertFails(t, filesToProcess, expectedErr, opts)
+		})
+		t.Run("is not a tuple", func(t *testing.T) {
+			schemaYAML := `#@data/values-schema
+---
+#@schema/examples "example value"
+key: val
+`
+			expectedErr := `
+Invalid schema
+==============
+
+syntax error in @schema/examples annotation
+schema.yml:
+    |
+  3 | #@schema/examples "example value"
+  4 | key: val
+    |
+
+    = found: non tuple in @schema/examples (by schema.yml:3)
+    = expected: tuple with optional string description and required example value
+    = hint: use a trailing comma to construct tuple with a single value. e.g. ('example value',)
+`
+
+			filesToProcess := files.NewSortedFiles([]*files.File{
+				files.MustNewFileFromSource(files.NewBytesSource("schema.yml", []byte(schemaYAML))),
+			})
+
+			assertFails(t, filesToProcess, expectedErr, opts)
+		})
+		t.Run("is an empty tuple", func(t *testing.T) {
+			schemaYAML := `#@data/values-schema
+---
+#@schema/examples ()
+key: val
+`
+			expectedErr := `
+Invalid schema
+==============
+
+syntax error in @schema/examples annotation
+schema.yml:
+    |
+  3 | #@schema/examples ()
+  4 | key: val
+    |
+
+    = found: empty tuple in @schema/examples (by schema.yml:3)
+    = expected: tuple with optional string description and required example value
+`
+
+			filesToProcess := files.NewSortedFiles([]*files.File{
+				files.MustNewFileFromSource(files.NewBytesSource("schema.yml", []byte(schemaYAML))),
+			})
+
+			assertFails(t, filesToProcess, expectedErr, opts)
+		})
+		t.Run("has more than two args in an example", func(t *testing.T) {
+			schemaYAML := `#@data/values-schema
+---
+#@schema/examples ("key example", "val", "value")
+key: val
+`
+			expectedErr := `
+Invalid schema
+==============
+
+syntax error in @schema/examples annotation
+schema.yml:
+    |
+  3 | #@schema/examples ("key example", "val", "value")
+  4 | key: val
+    |
+
+    = found: 3 arguments in @schema/examples (by schema.yml:3)
+    = expected: no more than 2 arguments per tuple. e.g. @schema/examples ('description', exampleValue())
+`
```

Comment:
```suggestion
    = found: 3-tuple argument in @schema/examples (by schema.yml:3)
    = expected: 2-tuple containing description (string) and example value (of expected type)
`
```

nit: pedantically, it's the requisite one argument, but has the wrong cardinality.